### ISSUE TITLE
Total balance increased after repay with collateral

### ISSUE THREAD

**Issue by coolrobiul** [script3/yieldblox-beta#349]
My lended value was 2.93M
Borrowed balance was 2.37m+
![Screenshot_2021-10-16-06-27-03-53_4d38fce200f96aeac5e860e739312e76](https://user-images.githubusercontent.com/59414838/137742091-023a6103-bb0b-47c9-abca-29e1797a304d.jpg)

My HF fall down to 1.02 
So I decided to repay with collateral
After successfully repayed
I lended again & now my lended value 5m+ 
And borrowed value 2.26m
How my balance increased??

![Screenshot_2021-10-18-19-16-00-12_4d38fce200f96aeac5e860e739312e76](https://user-images.githubusercontent.com/59414838/137742738-df6499d9-b656-4d92-967d-433491cbb3bf.jpg)

**Comment by mootz12** [comment]
It appears you doubled the amount of ETH you were lending, which amounts to about ~$2.1M.

Is this not expected?

**Comment by coolrobiul** [comment]
No, it's not expected i think
Not only double
I can borrow+ lend more with my health factor

Question is where I got balance 

Now its already 3m+ & can do more
![Screenshot_2021-10-18-20-02-31-63_4d38fce200f96aeac5e860e739312e76](https://user-images.githubusercontent.com/59414838/137746740-724e772f-5716-468f-b903-0cf4ac368721.jpg)

**Comment by coolrobiul** [comment]
After repay there are still some eth  in my wallet & I lended them

After repay I should able to lend+ borrow less than before

But now I'm able to lend+borrow even more after repay

**Comment by starlog20** [comment]
did your lent jumping after you lend some assets?
If yes, probably similar with my case here #205
i also repayed and did lent after too but i dunno why my lent is jumping after my deposit is successed. But i'm not try to borrow yet in past because try to get proof. but after i get it, my balance immediately back to normal.
Still not sure what happen exactly

**Comment by mootz12** [comment]
@coolrobiul, at the time, was EURT your primary borrow?

**Comment by coolrobiul** [comment]
It seems to your case ur balance went back to normal again after some time
But For my case it wasn't jump & not back to normal

There was still same balance in Eth wich I repayed with collateral...May be balance not deducted after repay with collateral So i was able to lend it again

**Comment by coolrobiul** [comment]
@mootz12 yes at that time EURT was my primary borrow
& I repayed with ETH collateral balance

**Comment by starlog20** [comment]
is it possible if the repayed balance clawback to debitor balance when error or failed on smartcontract happen?

**Comment by mootz12** [comment]
Nope, all turrets must reach consensus if any XDR is going to be generated. Having trouble finding something individually wrong with any particular transaction.  Going to keep looking, tho - thanks for the info!
